src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyClaimantJudgmentVariedDeterminationOfMeansNotificationHandler.java

```diff
@@ -0,0 +1,106 @@
+import uk.gov.hmcts.reform.civil.service.OrganisationService;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.NOTIFY_CLAIMANT_JUDGMENT_VARIED_DETERMINATION_OF_MEANS;
+import static uk.gov.hmcts.reform.civil.enums.MultiPartyScenario.ONE_V_ONE;
+import static uk.gov.hmcts.reform.civil.enums.MultiPartyScenario.TWO_V_ONE;
+import static uk.gov.hmcts.reform.civil.enums.MultiPartyScenario.getMultiPartyScenario;
+import static uk.gov.hmcts.reform.civil.utils.PartyUtils.getPartyNameBasedOnType;
+
+@Service
+@RequiredArgsConstructor
+public class NotifyClaimantJudgmentVariedDeterminationOfMeansNotificationHandler extends CallbackHandler
+    implements NotificationData {
+
+    private static final List<CaseEvent> EVENTS = List.of(NOTIFY_CLAIMANT_JUDGMENT_VARIED_DETERMINATION_OF_MEANS);
+    public static final String TASK_ID = "NotifyClaimantJudgmentVariedDeterminationOfMeans";
+    private static final String REFERENCE_TEMPLATE =
+        "claimant-judgment-varied-determination-of-means-%s";
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+    private final OrganisationService organisationService;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
+            callbackKey(ABOUT_TO_SUBMIT),
+            this::notifyClaimantJudgmentVariedDeterminationOfMeans
+        );
+    }
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse notifyClaimantJudgmentVariedDeterminationOfMeans(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        if (caseData.getApplicantSolicitor1UserDetails().getEmail() != null) {
+            notificationService.sendMail(
+                caseData.getApplicantSolicitor1UserDetails().getEmail(),
+                getTemplate(),
+                addProperties(caseData),
+                getReferenceTemplate(caseData)
+            );
+        }
+
+        return AboutToStartOrSubmitCallbackResponse.builder().build();
+    }
+
+    @Override
+    public Map<String, String> addProperties(CaseData caseData) {
+        String defendantName = "";
+        if (getMultiPartyScenario(caseData).equals(ONE_V_ONE)
+            || getMultiPartyScenario(caseData).equals(TWO_V_ONE)) {
+            defendantName = getPartyNameBasedOnType(caseData.getRespondent1());
+        } else {
+            defendantName = getPartyNameBasedOnType(caseData.getRespondent1())
+                .concat(" and ")
+                .concat(getPartyNameBasedOnType(caseData.getRespondent2()));
+        }
```

Comment:
Can we extract the line 76-84 to a method for better readability. Also this is also something we all are doing in Notification classes. Think, this can be also moved to Utils and reused.